README.md

```diff
@@ -59,6 +69,8 @@ And then access Kibana UI by hitting [http://localhost:5601](http://localhost:56
 
 *NOTE*: You'll need to inject data into Logstash before being able to configure a Logstash index pattern in Kibana. Then all you should have to do is to hit the create button.
 
+By default, only the `logspout-*` index is automatically created to provide out-of-the-box indexing of Docker Log messages.
```

Comment:
I think he did this in error (if you notice the Create index section it's correct)